ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/streaming/StreamedQueryResourceTest.java

```diff
@@ -304,44 +302,24 @@ public void shouldNotCreateExternalClientsForPullQuery() {
   }
 
   @Test
-  public void shouldThrowExceptionForPullQueryIfValidating() {
-    // When:
-    final Response response = testResource.streamQuery(
-        securityContext,
-        new KsqlRequest(PULL_QUERY_STRING, Collections.emptyMap(), null)
-    );
-
-    // Then:
-    assertThat(response.getStatus(), is(Errors.badRequest("").getStatus()));
-    assertThat(response.getEntity(), is(instanceOf(KsqlErrorMessage.class)));
-    final KsqlErrorMessage expectedEntity = (KsqlErrorMessage) response.getEntity();
-    assertThat(
-        expectedEntity.getMessage(),
-        containsString(KsqlConfig.KSQL_PULL_QUERIES_SKIP_ACCESS_VALIDATOR_CONFIG)
-    );
-  }
-
-  @Test
-  public void shouldPassCheckForPullQueryIfNotValidating() {
```

Comment:
is there a test to ensure the backend auth is not invoked if validation is not required? i.e non-rbac/acl case with insecure clusters

Reply:
Those tests are executed in other classes, like in the cache. I don't think we need that test case in this place.
I will merge this PR and add more tests if I find more.